.github/workflows/ci_BE.yml

```diff
@@ -34,7 +34,8 @@ jobs:
           echo "MONGO_INITDB_ROOT_USERNAME=github-ci" >> env/mongodb.env
           echo "MONGO_INITDB_ROOT_PASSWORD=github-ci" >> env/mongodb.env
           echo "REACT_APP_BACKEND_API_BASE_URL=http://localhost:8000/" >> env/frontend.env
-          docker compose run init-mongo
+          docker compose up -d backend
+          sleep 60
```

Comment:
bruh

Reply:
how else do i wait for the backend to start thooooo

Reply:
ok i came up with a less dumb solution